sections/standard.adoc

```diff
@@ -271,6 +284,51 @@ result manifest in absence of `key` member.
 <10> Optional string containing key of result manifest member for extraction. This allows mapping from a result manifest
 member key that differs from the value of `name` member.
 
+[[settings-section, Settings Object]]
+====== Settings Object
+The Settings object is the member responsible for indicating all content not related to data that is needed for the
+Job to run.
+
+* The Settings object MUST have a member `name`. The member's value is a string that correlates to referenced setting
+injections elseware in the Interface.
+* The Settings object MUST have a member `value`. The member's value is a string that indicates the desired value to
+associate with the setting object.
+is required by the job.
+
+The following annotated snippet provides quick reference to the use of the Settings object:
+
+[source,javascript]
+----
+{
+  "name": "SETTING1", // <1>
+  "value": "5432" // <2>
+}
+----
+<1> Required string containing the name to be used to lookup uses in the Interface.
+<2> Required string indicating if the value to associate with the setting name.
+
+[[env_vars-section, Environment Variables Object]]
+====== Settings Object
+The Environment Variables object is the member responsible for defining any values that need to be set as environment
+variables in that runtime that will be executing the job.
+
+* The env_vars object MAY have a member `name`. The member's value is an array of objects defined in Files Elements
+sub-section.
+* The env_vars object MAY have a member `value`. The member's value is an array of objects defined in JSON Elements
```

Comment:
Is the member's value actually an array of objects.  Isn't it just a value that is a case sensitive reference to a `Settings` object key?